style.css

```diff
@@ -1,132 +1,323 @@
-    padding: 0;
-    gap: 16px;
-    display: flex;
-    justify-content: flex-start;
-    flex-direction: row;
-    align-items: flex-start;
-    width: 117px;
-    height: 24px;
-    background: white;
-  }
-
-  nav a {
-    font-style: normal;
-    font-weight: 700;
-    font-size: 24px;
-    line-height: 20px;
-    text-decoration: none;
-    width: 250px;
-    height: 20px;
-    color: #6070ff;
-    display: flex;
-    background: white;
-  }
-
-  main {
-    background-image: url("./BG.jpg");
-    background-repeat: no-repeat;
-    background-size: cover;
-    background-position: center;
-    position: absolute;
-    top: 5%;
-    left: 0;
-    margin: 10px 0;
-    width: 100%;
-    display: flex;
-    border-bottom-left-radius: 62px;
-    border-bottom: 2px solid #e5e5e5;
-  }
-
-  section.main-section {
-    margin: 114px 24px;
-    background: white;
-  }
-}
-
-/* @media only screen and (min-width: 768px) {
-  nav img {
-    font-size: 15px;
-    width: 24px;
-    height: 24px;
-    margin: 0 10%;
-  }
-} */
+* {
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+}
+
+header nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header a {
+  margin: 0 12px;
+  font-size: 18px;
+}
+
+nav img {
+  font-size: 15px;
+  width: 24px;
+  height: 20px;
+  margin: 10px 24px;
+}
+
+h1.name {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 114px;
+}
+
+p.description {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 300px;
+}
+
+h3,
+.connect {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin: 12px;
+}
+
+.connect {
+  margin: 0;
+}
+
+ul.links {
+  padding: 0;
+  gap: 16px;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-items: flex-start;
+  width: 117px;
+  height: 24px;
+  margin-right: 12px;
+  margin-left: 12px;
+}
+
+.link2 {
+  margin-top: 50px;
+}
+
+nav a {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 20px;
+  text-decoration: none;
+  width: 250px;
+  height: 20px;
+  color: #6070ff;
+  display: flex;
+}
+
+section.main-section {
+  margin: 10px 24px;
+}
+
+main {
+  background-image: url(./BG.jpg);
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 6%;
+  left: 0;
+  right: 0;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin: 0 24px;
+}
+
+.projects {
+  margin: 0 24px;
+  padding: 20px 0;
```

Comment:
- [ ] Please 🙏🏽 use `padding: 0;` to reduce the space between the projects as requested by the previous reviewer to be similar to the Figma template.